apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/_components/MetricsSummary/Charts/Numerical/CostOverResults.tsx

```diff
@@ -82,12 +82,24 @@ export function CostOverResultsChart({
               type: 'number',
               min: evaluationConfiguration.minValue,
               max: evaluationConfiguration.maxValue,
+              tickFormatter: (value) =>
+                Intl.NumberFormat('en-US', {
+                  style: 'decimal',
+                  maximumFractionDigits: 2,
+                }).format(Number(value)),
             },
             yAxis: {
               label: 'Average cost',
               type: 'number',
               min: 0,
               max: 'auto',
+              tickFormatter: (value) =>
+                Intl.NumberFormat('en-US', {
+                  style: 'currency',
+                  currency: 'USD',
+                  minimumFractionDigits: 0,
+                  maximumFractionDigits: 6,
+                }).format(Number(value)),
```

Comment:
Can we either align the text to the left, or use always the same number of digits? That way is way easier to compare decimal values at a glance.